tests/trainer/test_dataloaders.py

```diff
@@ -1077,7 +1077,7 @@ def test_dataloaders_load_every_epoch_no_sanity_check(tmpdir):
         limit_train_batches=0.3,
         limit_val_batches=0.3,
         num_sanity_val_steps=0,
-        reload_dataloaders_every_epoch=True,
+        reload_dataloaders_every_n_epoch=True,
```

Comment:
Can you add a test for it ?

Reply:
Okay. There was already a test for when `reload_dataloaders_every_n_epochs=True`. I added one for `reload_dataloaders_every_n_epochs` > 1